server/lib/template/partials/head.handlebars

```diff
@@ -1,8 +1,10 @@
 <!-- this partial requires binding the variable 'title' -->
 <head>
   <title>{{translate title}}</title>
+  <base href="/"></base>
```

Comment:
As discussed; this base has no function for now. This could be removed until required.